named_entity_recognition/documentation/step-4-custom-prediction-routine.md

```diff
@@ -0,0 +1,41 @@
+# Custom prediction routine
+
+Allow us to determine which code runs when sending a prediction request.
```

Comment:
How about replacing this sentence with, "Custom prediction routines allow us to specify additional code that runs with every prediction request." You could remove the last sentence since it says essentially the same thing.

Reply:
✓ done